### DISCUSSION TITLE
Does anyone know why http://ui.tabix.io/ is not accessible？

### DISCUSSION THREAD

**Discussion by Tiakon** [tabixio/tabix#262] [Q&A] [answered]
I used an embedded Tabix link ​in clickhouse as a visual interface for editing sql.  Now it is inaccessible.  I wonder what happened to it and whether it will recover？

**Comment by isublimity** [comment, marked as answer]
Lost server with all data -( 
I'm waiting for the hosting to get a backup if it succeeds.

**Reply by Tiakon** [reply]
Oh no, sorry to hear that! Hoping for the best and keeping my fingers crossed that the backup works out.
